source/tests/system/nidcpower_driver_api_tests.cpp

```diff
@@ -181,6 +181,18 @@ class NiDCPowerDriverApiTest : public ::testing::Test {
     EXPECT_EQ(kdcpowerDriverApiSuccess, response.status());
   }
 
+  void configure_output_function(const char* channel_name, ViInt32 function)
+  {
+    ::grpc::ClientContext context;
+    dcpower::ConfigureOutputFunctionRequest request;
+    request.mutable_vi()->set_id(GetSessionId());
+    request.set_channel_name(channel_name);
+    request.set_function(function);
+    dcpower::ConfigureOutputFunctionResponse response;
+
+    ::grpc::Status status = GetStub()->ConfigureOutputFunction(&context, request, &response);
```

Comment:
@ni-pkhare , you need to add EXPECT_EQ statements here right for asserting that status is OK and call is successful? All other helpers have that

Reply:
Done. Forgot to do it earlier.